programs/token-voter/src/governance.rs

```diff
@@ -0,0 +1,217 @@
+/// A macro is exposed so that we can embed the program ID.
+#[macro_export]
+macro_rules! vote_weight_record {
```

Comment:
they are not exported in the IDL as an account (only Registrar and Voter are there). but the discriminator is included in the instructions and the types as well.

Would you like me to make it a regular state account instead of a macro?